docs/source/markdown/podman-pod-create.1.md

```diff
@@ -141,7 +141,8 @@ Set network mode for the pod. Supported values are:
 
 #### **--network-alias**=strings
 
-Add a DNS alias for the container. When the container is joined to a CNI network with support for the dnsname plugin, the container will be accessible through this name from other containers in the network.
+Add a DNS alias for the container. When the container is joined to a CNI network with support for the dnsname plugin, the container will be accessible through this name from other containers in the network.  NOTE: A container will only have access to aliases on the first network that it joins.  This is a limitation
+that will be removed in a later release.
```

Comment:
Does this line break effect man page generation?

Reply:
No, it just ignores the line break, just verified.